documentation/manual/src/main/asciidoc/en-US/modules/mongodb.asciidoc

```diff
@@ -118,6 +118,16 @@ Possible strategies are (values of the `org.hibernate.ogm.datastore.mongodb.opti
 
 * `GLOBAL_COLLECTION` (default): stores the association information in a unique MongoDB collection for all associations
 * `COLLECTION_PER_ASSOCIATION` stores the association in a dedicated MongoDB collection per association
+
+hibernate.ogm.datastore.document.map_storage::
+Defines the way OGM stores the contents of map-typed associations in MongoDB.
+The following two strategies exist (values of the `org.hibernate.ogm.datastore.document.options.MapStorageType` enum):
+
+* `BY_KEY`: map-typed associations with a single key column which is of type `String` will be stored as a sub-document,
+organized by the given key; Not applicable for other types of key columns, in which case always `AS_LIST` will be used
```

Comment:
That bridge class (hierarchy) is already there, `GridType` :) Question is whether we really want it or not.